data/campaigns/Dead_Water/scenarios/04_Slavers.cfg

```diff
@@ -499,6 +499,7 @@
 #endif
                 [message]
                     speaker=narrator
+					#po: "to hand" is an expression that means "nearby".
```

Comment:
Maybe trivial, but indenting doesn't look right here.

On the text itself, is 'to hand' really a valid expression, archaic or otherwise? In other contexts I might suggest a change to 'if no weapons were at hand'.

Reply:
Interesting. Thanks, I don't think I've heard of this before and must have missed it (or it wasn't there) when I last played the campaign some years ago.

Reply:
it was tabs vs spaces. Forgot which IDE I was using (some IDEs automatically convert tabs to spaces). Fixed,